path.go

```diff
@@ -86,6 +87,21 @@ func NewPathFromXYData(data [][2]float64) *Path {
 	return p
 }
 
+// NewPathFromFlatXYData creates a path from a slice of float64 values
+// representing horizontal, vertical type data, for example lng/lat values from geojson.
```

Comment:
please update this comment, geojson does not have flat coordinates like this func expects.